mutiny-core/src/storage.rs

```diff
@@ -549,15 +550,26 @@ pub trait MutinyStorage: Clone + Sized + Send + Sync + 'static {
         self.set_data(FIRST_SYNC_KEY.to_string(), true, None)
     }
 
-    fn get_dm_sync_time(&self) -> Result<Option<u64>, MutinyError> {
-        self.get_data(LAST_DM_SYNC_TIME_KEY)
+    fn get_dm_sync_time(&self, is_hermes: bool) -> Result<Option<u64>, MutinyError> {
+        let key = if is_hermes {
+            LAST_HERMES_SYNC_TIME_KEY
+        } else {
+            LAST_DM_SYNC_TIME_KEY
+        };
+        self.get_data(key)
     }
 
-    fn set_dm_sync_time(&self, time: u64) -> Result<(), MutinyError> {
+    fn set_dm_sync_time(&self, time: u64, is_hermes: bool) -> Result<(), MutinyError> {
```

Comment:
it would be better to just make this an alternative key anyways. If we do more things with alternative nostr key DMs, which I imagine we will soon, then this boolean won't make very much sense.

Reply:
Yeah I figured we'd just switch to an enum, I can do that now I guess

Reply:
That's fine, no need to block on it.